modules/nw-ovn-kubernetes-ansible-migration-about.adoc

```diff
@@ -0,0 +1,145 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc
+
+[id="nw-ovn-kubernetes-ansible-migration-about_{context}"]
+= Using the Ansible collection to migrate to the OVN-Kubernetes network plugin
+
+As a cluster administrator, you can use an Ansible collection, `network.offline_migration_sdn_to_ovnk`, to migrate from the OpenShift SDN Container Network Interface (CNI) network plugin to the OVN-Kubernetes plugin for your cluster. The Ansible collection includes the following playbooks:
+
+* `playbooks/playbook-migration.yml` that migrates from the OpenShift SDN CNI plugin to the OVN-Kubernetes plugin.
+* `playbooks/playbook-rollback.yml` that rolls back from the OVN-Kubernetes plugin to the OpenShift SDN CNI plugin.
+
+.Prerequisites
+
+* You installed the `python3` packages, minimum version 3.10.
+* You installed the ` jmespath`, `jq`, and `make` packages.
+* You installed Docker.
+* You logged in to the {hybrid-console} and opened the link:https://console.redhat.com/ansible/ansible-dashboard[Ansible Automation Platfor] web console.
+* You have access to the `registry.ci.openshift.org` container image registry that is used by the {product-title} Continuous Integration (CI) functionality.
```

Comment:
@miheer . If a customer cannot access this registry, they need to run the Dockerfile.debug to build the image?